requirements.txt

```diff
@@ -19,3 +19,4 @@ selenium==3.5.0
 django-suit==0.2.18
 six>=1.10.0
 django-axes==2.3.3
+django-celery-results
```

Comment:
@childers Although I am not quite sure what `Django ORM/Cache` means, I think we don't need this. I compared the counterpart in celery documentation:
- 4.1: http://docs.celeryproject.org/en/v4.1.0/django/first-steps-with-django.html#django-celery-results-using-the-django-orm-cache-as-a-result-backend
- 3.1: http://docs.celeryproject.org/en/3.1/django/first-steps-with-django.html#using-the-django-orm-cache-as-a-result-backend

If we need this, we should install `django-celery` before.

Reply:
Oh, I finally figured out that `amqp` result backend can't be used anymore ...
@childers Do you need to run `python manage.py migrate` before start using this?